zone_failure/failure.sh

```diff
@@ -31,7 +31,7 @@ INSTANCE_NAME=$(hostname)
 STATE="healthy"
 while true; do
   if [[ "$ZONE" = "$(GetMetadata $PROJECT_METADATA_URL/failed_zone)" ]] && \
-     [[ "$INSTANCE_NAME" =~ "$(GetMetadata $PROJECT_METADATA_URL/failed_instance_names)" ]]; then
+     [[ "$INSTANCE_NAME" =~ $(GetMetadata $PROJECT_METADATA_URL/failed_instance_names) ]]; then
     if [[ "$STATE" = "healthy" ]]; then
```

Comment:
```
In ./zone_failure/failure.sh line 34:
     [[ "$INSTANCE_NAME" =~ "$(GetMetadata $PROJECT_METADATA_URL/failed_instance_names)" ]]; then
                            ^-- SC2076: Don't quote rhs of =~, it'll match literally rather than as a regex.
```

Or did you mean for this to be equality?